golang/cosmos/app/upgrade.go

```diff
@@ -106,19 +101,16 @@ func buildProposalStepWithArgs(moduleName string, entrypoint string, opts map[st
 }
 
 func getVariantFromUpgradeName(upgradeName string) string {
-	switch upgradeName {
-	case "UNRELEASED_A3P_INTEGRATION":
+    switch upgradeName {
+    case "agoric-upgrade-18":
+        return "MAINNET"
```

Comment:
So this is the reason why tests are failing, right?
@mujahidkay

It should be a3p plan variant instead of MAINNET. And you will be updating the planName in a3p proposal json file as well. Did I get it right?